pom.xml

```diff
@@ -19,23 +19,23 @@
   <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>2.3.8.RELEASE</version>
+    <version>2.3.11.RELEASE</version>
     <relativePath /> <!-- lookup parent from repository -->
   </parent>
 
   <properties>
-    <tomcat.version>9.0.43</tomcat.version>
-    <com.fasterxml.jackson.version>2.11.4</com.fasterxml.jackson.version>
```

Comment:
During 1.10.1 release, Jackson version 2.12.* wasn't compatible with AIO. Can we check whether this PR is not breaking AIO flow?
Check PR #53.

Reply:
Noted, reverted back to 2.11.4.